instructions/docs/add-the-encryption-sdk.md

```diff
@@ -106,20 +106,19 @@ const Body = fileStream.pipe(encryptStream(encryptKeyring));
 
 ```Python
 # Edit src/document_bucket/api.py
-
-def store(self, data: bytes, context: Dict[str, str] = {}) -> PointerItem:
+# Find the store function and edit it...
```

Comment:
consider "Find the `store` function and edit it to add the following parameters:"

Reply:
Yep!